Changes

```diff
@@ -1,5 +1,10 @@
 Revision history for Perl module Perl::Critic
 
+NEXT
+
+    [New Policies]
+    * Added a policy: ControlStructures::ProhibitYadaOperator.
```

Comment:
Please add the description of the policy here (from the NAME section of the policy POD). Thanks